### PR TITLE
Development

### PR DESCRIPTION
**Pull request by SolivanLau**

### CODE REVIEW

File: public/logo192.png
Comment (Pedram22) [inline public/logo192.png:1]: Great idea changing the favicon!

File: src/Components/ListsInfo.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            params: {
+                apiKey: process.env.REACT_APP_API_KEY_ONE,
+                query: userSearch,
+                number: 20,
+                metaInformation: true
+            }
+        }).then((apiData) => {
+            const autoCompleteData = apiData.data;
+            setAutocompleteArr(autoCompleteData)
+            // console.log(autoCompleteArr)
```
Comment (Pedram22) [inline src/Components/ListsInfo.js:90]: console.log is commented out, but might want to remove entirely.

File: src/assets/appBG.svg
Comment (Pedram22) [inline src/assets/appBG.svg:1]: Awesome background!

File: src/Components/FoodItem.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                                Enter a date <span className="attention">after</span> today's date... or you might have some stinky {name}
+                            </p> : null}
+
+                    </div>
+                </>
+            }
+
+            {/* REMOVE FOOD ITEM BUTTON */}
+            <button className="symbolBtn removeBtn" onClick={handleRemove}>
+                <span className='sr-only'>Delete {name} from {currentMode.title}</span>
```
Comment (Pedram22) [inline src/Components/FoodItem.js]: Great use of code for accessibility!

File: public/index.html [hunk 3]
```diff
@@ -6,8 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="description"
-      content="Web site created using create-react-app"
+      name="Stock My Fridge"
+      content="This site uses fundamental react hooks in combination with SCSS and React Router Dom packages."
```
Comment (Pedram22) [inline public/index.html:10]: Will have to look into changing these in my app!

Comment (DevinD55) [inline public/index.html:10]: This is small stuff but really cool catch!

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,29 @@
...
+        <div className="wrapper">
+          <h1>Stock My Fridge!</h1>
+        </div>
       </header>
+
+      <ListsInfo />
+
+      <footer>
+        <div className="wrapper">
+          <p>Made by Solivan Lau at JunoCollege 2023</p>
```
Comment (DevinD55) [inline src/App.js:20]: Consider linking this to your portfolio or LinkedIn as an anchor tag

**Review by Pedram22 — APPROVED** [review]

File: src/App.scss
Comment (DevinD55) [inline src/App.scss:1]: nice comments to keep it clean and easy to read

File: src/Components/FoodItem.js
Comment (DevinD55) [inline src/Components/FoodItem.js:1]: This looks amazing! Really detailed use of useState for all of these variables

File: src/Components/FoodItem.js [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+            // miliseconds divided by calc for milisec in a day
+            const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24))
+
+            if (diffDays < 0) {
+
+                console.log('ERROR - will NOT be sent to DB')
+                setExpError(true)
+            } else {
+
+                console.log('passed vibe check')
```
Comment (DevinD55) [inline src/Components/FoodItem.js:78]: I'm sure you're leaving for review, but remove console logs before submitting

File: src/Components/FoodItem.js [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+        const removeExpError = () => {
+            if (expError === true) {
+                setExpError(false)
+                // // ref to specific firebase item
+                // const foodItemExpRef = ref(database, `fridgeList/${fbId}`);
+
+                // // creating empty value for expdate property
+                // const date = { expDate: '' };
+                // // take user input and update on firebase 
+                // update(foodItemExpRef, date)
```
Comment (DevinD55) [inline src/Components/FoodItem.js]: remove commented out code

File: src/Components/FoodItem.js [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+
+
+
+                        {/* ERROR MSG IF user enters date before today's date */}
+
+                        {expError === true ?
+
+                            <p className='errorMsg'>
+                                Enter a date <span className="attention">after</span> today's date... or you might have some stinky {name}
```
Comment (DevinD55) [inline src/Components/FoodItem.js:173]: Crazy error handling, super detailed stuff!

File: src/Components/Form.js
Comment (DevinD55) [inline src/Components/Form.js:1]: There's a lot in here and I can't find much to comment on. Overall really impressive

File: src/Components/Fridge.js [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+// database details and reference
+const database = getDatabase(firebaseDB)
+const dbFridgeRef = ref(database, `fridgeList/`)
+
+
+
+const Fridge = ({ userSearch, searchError, autoCompleteArr, pushFoodtoDB, handleChange, handleSuggest, setSearchError }) => {
+
+    const fridgeMode = {
```
Comment (DevinD55) [inline src/Components/Fridge.js:19]: opposite of sicko mode

File: src/Components/Fridge.js
Comment (DevinD55) [inline src/Components/Fridge.js:1]: great comments throughout explaining function of each code block

File: src/Components/ListsInfo.js
Comment (DevinD55) [inline src/Components/ListsInfo.js:1]: Nice use of turnerys here! Mixing everything we've learned together is really impressive

File: src/scssPartials/_foodItem.scss
Comment (DevinD55) [inline src/scssPartials/_foodItem.scss:1]: I'm running out of time, so not able to review completely, but the CSS looks great on the page!

**Review by DevinD55 — COMMENTED** [review]
Really awesome job! I'm really impressed with the details. Tiny things like linking to your portfolio would be great.

**Pull request merged**